apps/judicial-system/web/src/types/index.ts

```diff
@@ -14,11 +14,11 @@ export enum Sections {
 }
 
 export enum ProsecutorSubsections {
-  CUSTODY_REQUEST_STEP_ONE = 0,
-  CUSTODY_REQUEST_STEP_TWO = 1,
-  CUSTODY_REQUEST_STEP_THREE = 2,
-  CUSTODY_REQUEST_STEP_FOUR = 3,
-  CUSTODY_REQUEST_STEP_FIVE = 4,
+  CUSTODY_CASE_STEP_ONE = 0,
```

Comment:
Should these be called `RESTRICTION_CASE_???`?

If they are use for both travel bans and custodies, then I think that would be appropriate.

Reply:
These enum values are both used for `InvestigationCases` and `RestrictionCases`. I think we can just rename this to `CASE_STEP_ONE` etc.

Reply:
Resolved in [876ae05](https://github.com/island-is/island.is/pull/6952/commits/876ae05d0f90f837dd6be3b4e1fc8a3ae2f65559)